core/vm/contracts_suave.go

```diff
@@ -123,6 +124,8 @@ func (c *confStoreStore) runImpl(suaveContext *SuaveContext, bidId suave.BidId,
 		return suave.ErrBidNotFound
 	}
 
+	log.Info("confStoreStore", "bidId", bidId, "key", key)
```

Comment:
probably not needed

Reply:
We do have the same log for retrieve, do you want to remove that one too?

Reply:
I think so